packages/core/src/state-management/repository.ts

```diff
@@ -614,10 +614,16 @@ export class Repository {
         return false
       }
       case AnchorRequestStatusName.COMPLETED: {
-        if (anchorEvent.cid.equals(state$.tip)) {
+        const anchorCommitCID = anchorEvent.witnessCar.roots[0]
+        await this._handleAnchorCommit(state$, anchorEvent.cid, anchorEvent.witnessCar)
```

Comment:
do we expect `anchorEvent.witnessCar.roots[0]` and  `anchorEvent.cid` to be different?

Reply:
AH i see thanks